docs/DeveloperGuide.md

```diff
@@ -213,9 +213,45 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 _{more aspects and alternatives to be added}_
 
-### \[Proposed\] Data archiving
 
-_{Explain here how the data archiving feature will be implemented}_
+### \[Proposed\] Deck System
+This feature would allow the user to create multiple lists of entries rather than having 
+all translations together in the same list. This means that the user could have different decks
+for different languages or even multiple decks for the same languages. 
+
+
+_{Feature will be added in v1.2}_
+
+### \[Proposed\] Flashcard System
+
+The flashcard system would allow the user to choose to practice in whichever deck
+he wishes.
+
+The methodology behind GreenTea's flashcard system will be based on the Leitner System (https://en.wikipedia.org/wiki/Leitner_system)
+To put it simply, it is a proven quizzing system that increases the user's rate of learning by 
```

Comment:
I think the 'to put it simply' can be removed. Maybe something like " It is a proven quizzing system that increases the user's rate of learning by using spaced repetition. Flashcards are sorted based on the user's ability to answer them.

Reply:
Noted, will update this

Reply:
Updated!